### PR TITLE
windows CI: Fix path after platform update

### LINKED ISSUE #18607: CI failing in windows (Unbound value Glib.Io.channel_of_descr_socket)
eg https://github.com/SkySkimmer/coq/actions/runs/7742277991/job/21111031505
~~~
<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
-> installed conf-findutils.1
-> installed conf-gmp.4
-> installed conf-pkg-config.3
-> installed conf-adwaita-icon-theme.2
-> installed conf-cairo.1
-> installed conf-gtk3.18
-> installed dep-glib-compiled-schemas.1.0
-> installed ocamlfind.1.9.5~relocatable
-> installed zarith.1.13
-> installed dune.3.13.0
-> installed camlp-streams.5.0.1
-> installed csexp.1.5.2
-> installed dune-configurator.3.13.0
-> installed cairo2.0.6.4
-> installed dep-gtksourceview3.24.11
-> installed conf-gtksourceview3.0+2
-> installed lablgtk3.3.1.4
-> installed lablgtk3-sourceview3.3.1.4
-> installed coq-core.dev
-> installed coqide-server.dev
Error:  The compilation of coqide failed at "C:\\ci\\cygwin64\\home\\runneradmin\\.opam\\CP.2023.03+b1~dev\\bin\\dune.exe build -p coqide -j 2 @install".
-> installed coq-stdlib.dev
-> installed coq.dev

#=== ERROR while compiling coqide.dev =========================================#
[...]

# (cd _build/default && C:\ci\cygwin64\home\runneradmin\.opam\CP.2023.03+b1~dev\bin\ocamlopt.opt.exe -w -40 -g -O3 -unbox-closures -I ide/coqide/.coqide_gui.objs/byte -I ide/coqide/.coqide_gui.objs/native -I C:/ci/cygwin64/home/runneradmin/.opam/CP.2023.03+b1~dev/lib/ocaml\threads -I C:\ci\cygwin64\home\runneradmin\.opam\CP.2023.03+b1~dev\lib\cairo2 -I C:\ci\cygwin64\home\runneradmin\.opam\CP.2[...]
# File "ide/coqide/coqide_WIN32.ml.in", line 51, characters 37-68:
# Error: Unbound value Glib.Io.channel_of_descr_socket
~~~
I see no channel_of_descr_socket  in https://github.com/garrigue/lablgtk/blob/lablgtk3/src/glib.mli (last changed 7 years ago) so I don't understand where it was supposed to come from but it did work in the past.

lablgtk3 3.1.4 is very recent so may be the cause but how?

cc @garrigue

### PR DESCRIPTION
**Pull request by SkySkimmer**
Fix https://github.com/coq/coq/issues/18607

### CODE REVIEW

**Review by MSoegtropIMC — APPROVED** [review]
LGTM to adjust to the path changes in Coq Platform

**Comment by MSoegtropIMC** [comment]
@SkySkimmer : sorry, on second though you also need to change
```
%BASH% --login -c "pwd && ls -la && cd platform
```
The `cd platform` needs to be `cd /platform`

File: dev/ci/platform/coq-pf-04-installer.bat [hunk 1]
```diff
@@ -17,7 +17,7 @@ REM mess in terms of using the GITHUB_WORKSPACE sometimes, and the
 REM CYGWIN home some others. I use the path here directly as to avoid
 REM issues with quoting, which in the previous script required some
 REM really obscure code.
-COPY /v /b %CYGROOT%\home\runneradmin\platform\windows_installer\*.exe %GITHUB_WORKSPACE%\artifacts || GOTO ErrorExit
+COPY /v /b %CYGROOT%\platform\windows_installer\*.exe %GITHUB_WORKSPACE%\artifacts || GOTO ErrorExit
```
Comment (MSoegtropIMC) [inline dev/ci/platform/coq-pf-04-installer.bat:20]: In addtion the `cd platform` needs to be changed to `cd /platform`

**Review by MSoegtropIMC — CHANGES_REQUESTED** [review]

**Comment by SkySkimmer** [comment]
seems to work

**Review by MSoegtropIMC — APPROVED** [review]
LGTM

**Comment by MSoegtropIMC** [comment]
> seems to work

Yes, I think you already changed it, while I was doing my comment. It looks good now.

**Comment by SkySkimmer** [comment]
merge?

**Comment by MSoegtropIMC** [comment]
@coqbot merge now

**Pull request merged**